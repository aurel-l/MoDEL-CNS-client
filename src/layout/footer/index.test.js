import React from 'react';
import { shallow } from 'enzyme';

import Footer from '.';

describe('<Footer />', () => {
  it('should render', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
