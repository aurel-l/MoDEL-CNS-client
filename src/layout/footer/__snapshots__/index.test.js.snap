// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<footer>
  <WithStyles(ForwardRef(Paper))>
    <WithStyles(ForwardRef(Toolbar))
      className="footer"
    >
      <a
        href="https://www.humanbrainproject.eu/"
        rel="noreferrer noopener"
        target="_blank"
      >
        <img
          alt="Human Brain Project website"
          height="100px"
          loading="lazy"
          src="logo-hbp.png"
        />
      </a>
      <a
        href="https://www.irbbarcelona.org/"
        rel="noreferrer noopener"
        target="_blank"
      >
        <img
          alt="IRB Barcelona website"
          height="100px"
          loading="lazy"
          src="logo-irb.png"
        />
      </a>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(Paper))>
</footer>
`;
